kotlin-native/runtime/src/main/kotlin/kotlin/native/Annotations.kt

```diff
@@ -116,6 +117,19 @@ public actual annotation class CName(actual val externName: String = "", actual
 @SinceKotlin("1.8")
 public actual annotation class ObjCName(actual val name: String = "", actual val swiftName: String = "", actual val exact: Boolean = false)
 
+/**
+ * Instructs the Kotlin compiler to generate a NS_ENUM typedef for the annotated enum class. The name of the generated type will
+ * be the name of the enum type with "Enum" appended. This name can be overridden with the "name" parameter, which is treated
+ * as an exact name. The enum literals will be prefixed with the type name, as they live in a global namespace.
+ * Swift naming will automatically remove these prefixes.
+ */
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.BINARY)
+@MustBeDocumented
```

Comment:
This makes the annotation visible in the generated documentation. Which might be nice.
But it also makes it appear in the generated Objective-C comment, which seems redundant. Let's remove it from there by adding the annotation to the stop list? https://github.com/JetBrains/kotlin/blob/b38b99cddc8ae684b4e16df45943c4851a71ba01/native/objcexport-header-generator/impl/k1/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportTranslator.kt#L774-L775